### ISSUE TITLE
Bullets are not shown on deferred creation (setTimeout, AJAX, etc)

### ISSUE THREAD

**Issue by artar94** [amcharts/amcharts4#3132]
If, after creating a series, you make a delay in the form of a SetTimeout or an AJAX request, then the created bullets will not be displayed until some manual update of the series. By manual update I mean `hide()`/`show()` or `deepInvalidate()`.

This clearly depends on the duration of the animation. If animation is on, `setTimeout(..., 50)` works fine, but `setTimeout(..., 2000)` doesn't. If animation is off, then even `setTimeout(..., 0)` doesn't work.

```js
let delay = 2000; // Not work
//delay = 500; // Work
setTimeout(() => {
  series.bullets.push(new am4charts.CircleBullet());
  //series.deepInvalidate(); // Partial fix
}, delay);
```
CodePen: https://jsfiddle.net/509wftL3/5/

We discovered this problem in Angular 11, but it is steadily repeated in vanilla JavaScript.

In general, our task is to download data for bullets from the server after only 1 series is visible. And we had a lot of problems with dynamically showing/hiding bullets. In one case, helped us `series.deepInvalidate()`, in another
```js
series.bullets.each(b => {
  b.show(0);
  b.hide(0);
}); 
```
But after some update everything broke again...

At the moment this does not work in `@amcharts/amcharts4` version `4.10.12`.

**Comment by martynasma** [comment]
No need to deepInvalidate the whole chart or series. Invalidating series would do the trick:

```JavaScript
setTimeout(() => {
  series.bullets.push(new am4charts.CircleBullet());
  series.invalidate();
}, delay);
```

That being said, it would make sense to do that automatically.

We'll implement it in the next version, and will notify you here when it's out.

Thanks!

**Comment by martynasma** [comment]
Fixed in [4.10.13](https://github.com/amcharts/amcharts4/releases/tag/4.10.13).

## [4.10.13] - 2020-12-18

### Added
- Export: PDF export options now support `scale`, `minWidth`, `minHeight`, `maxWidth`, and `maxHeight` options.

### Changed
- Setting `ignoreZeroValues = true` on `PieSeries`/`PyramidSeries` will now hide `null`-value slices, too.

### Fixed
- Inserting a new bullet into an initialized Series will now invalidate it cause the new bullet to appear immediately.
- Using `timezone` might shift axis label values by one day in some cases.
- `"i"` input date format was not properly parsing formats with no milliseconds and with timezone offset, e.g. (`"...+10:30"`).
- Export: `normalizeSVG()` method was producing wrong SVG width/height in output if scale parameter was not being passed in.
- RangeSelector plugin: "YTD" pre-defined period was not working properly with some data-grouping setups.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
